### PR TITLE
TSK-1507: Cleaned up some CodeSmells

### PR DESCRIPTION
**Pull request by tge20**
---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [ ] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [x] Commit message format → TSK-XXX: Your commit message.
- [x] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [x] SonarCloud analysis meets our standards
- [x] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [x] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [x] Readability

### CODE REVIEW

File: lib/taskana-core/src/main/java/pro/taskana/task/internal/AttachmentMapper.java [hunk 1]
```diff
@@ -125,13 +115,10 @@
...
-      value = {
-        @Result(property = "left", column = "ID"),
-        @Result(property = "right", column = "PLANNED")
-        //          ,  javaType = Instant.class,
-        //            typeHandler = InstantTypeHandler.class)
-      })
+  @Result(property = "left", column = "ID")
+  @Result(property = "right", column = "PLANNED")
+  //          ,  javaType = Instant.class,
+  //            typeHandler = InstantTypeHandler.class)
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/main/java/pro/taskana/task/internal/AttachmentMapper.java:121]: please delete this comment.

File: lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryMapper.java [hunk 2]
```diff
@@ -356,7 +350,7 @@
           + "<if test='attachmentReceivedIn !=null'> AND ( <foreach item='item' collection='attachmentReceivedIn' separator=' OR ' > ( <if test='item.begin!=null'> a.RECEIVED &gt;= #{item.begin} </if> <if test='item.begin!=null and item.end!=null'> AND </if><if test='item.end!=null'> a.RECEIVED &lt;=#{item.end} </if>)</foreach>)</if> "
           + "<if test='wildcardSearchValueLike != null and wildcardSearchFieldIn != null'>AND (<foreach item='item' collection='wildcardSearchFieldIn' separator=' OR '>t.${item} LIKE #{wildcardSearchValueLike}</foreach>)</if> "
           + "</where> "
-          + "), Y (ID, EXTERNAL_ID, CREATED, CLAIMED, COMPLETED, MODIFIED, PLANNED, DUE, NAME, CREATOR, DESCRIPTION, NOTE, PRIORITY, STATE, TCLASSIFICATION_KEY, "
```
Comment (mustaphazorgati) [inline lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryMapper.java:359]: was this on purpose?
Why did you change this?

Comment (tge20) [reply lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryMapper.java:359]: Oh no, mistake. Well seen 👍

**Comment by tge20** [comment]
https://sonarcloud.io/component_measures?branch=TSK-1507&id=tge20_taskana&metric=new_lines_to_cover&view=list

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]
You've introduced 10 Code Smells. Please fix them.

**Review by mustaphazorgati — DISMISSED** [review]

**Review by mustaphazorgati — DISMISSED** [review]

**Review by mustaphazorgati — APPROVED** [review]
